src/SFML/Graphics/Shape.cpp

```diff
@@ -148,6 +149,28 @@ FloatRect Shape::getGlobalBounds() const
 }
 
 
+////////////////////////////////////////////////////////////
+void Shape::batch(SpriteBatch& spriteBatch, float depth) const
+{
+    // Draw the shape itself
+    {
+        const std::size_t shapeCount = m_vertices.getVertexCount();
+        const Vertex*     shapePtr   = shapeCount > 0 ? &m_vertices[0] : nullptr;
```

Comment:
`m_vertices.data()` may be a more expressive way to get a pointer to the first element than taking the address of index zero.

Reply:
That's a `VertexArray` though. I don't think it has such a method.